widgetssdk/src/main/java/com/glia/widgets/call/CallView.kt

```diff
@@ -707,40 +719,26 @@ internal class CallView(
         }
     }
 
-    private fun showOptionsDialog(
```

Comment:
Why was this deleted, Was this unused or was this merged with some other dialog?

Reply:
This was used only once, so I decided to use function from `Dialogs` instead